third_party/2and3/werkzeug/contrib/profiler.pyi

```diff
@@ -1,15 +1,11 @@
-from typing import Any, Optional
+from typing import Any, AnyStr, Iterable, Optional, Protocol
 
-available: Any
+from ..middleware.profiler import *
 
-class MergeStream:
-    streams: Any
-    def __init__(self, *streams): ...
-    def write(self, data): ...
+class _Writable(Protocol):
+    def write(self, __s: AnyStr) -> Any: ...
```

Comment:
AnyStr doesn't make sense by itself. I guess you could make this protocol and MergeStream generic over AnyStr.